sdk/python/lib/pulumi/runtime/__init__.py

```diff
@@ -57,3 +57,48 @@
     register_resource_module,
     register_resource_package,
 )
+
+__all__ = [
+    # config
+    "set_config",
+    "set_all_config",
+    "get_config",
+    "get_config_env",
+    "get_config_env_key",
+
+    # mocks
+    "Mocks",
+    "set_mocks",
+    "test",
+
+    # settings
+    "Settings",
+    "configure",
+    "is_dry_run",
+    "reset_options",
+
+    # stack
+    "run_in_stack",
+    "get_root_resource",
+    "register_stack_transformation",
+
+    # invoke
+    "invoke",
+
+    # _json
+    "to_json",
+
+    # rpc
+    "ResourceModule",
+    "ResourcePackage",
+    "register_resource_module",
+    "register_resource_package",
+
+    # submodules
+    "config",
+    "mocks",
+    "settings",
+    "stack",
+    "invoke",
+    "rpc"
```

Comment:
Similarly, wondering if we can prune any of these (though, less of a concern here since these are already more "hidden" under `runtime`).

```suggestion
```

The only one I'm not sure about is `"rpc"` which may be worth still exporting because it has some utilities that aren't imported and then exported from `runtime`.